README.md

```diff
@@ -111,9 +111,9 @@ sudo apt-get install libomxil-bellagio-dev
 wget -O ffmpeg.tar.bz2 https://ffmpeg.org/releases/ffmpeg-snapshot-git.tar.bz2
 tar xvjf ffmpeg.tar.bz2
 cd ffmpeg
-sudo ./configure --arch=armel --target-os=linux --enable-gpl --enable-omx --enable-omx-rpi --enable-nonfree
+sudo ./configure --arch=arm --target-os=linux --enable-gpl --enable-omx --enable-omx-rpi --enable-nonfree
```

Comment:
Nice job on digging into FFmpeg's configure command. Wouldn't have even thought to look there. 👍